Appraisals

```diff
@@ -312,7 +312,6 @@ elsif Gem::Version.new('2.2.0') <= Gem::Version.new(RUBY_VERSION) \
     appraise 'rails5-postgres-redis-activesupport' do
       gem 'rails', '~> 5.2.1'
       gem 'pg', '< 1.0', platform: :ruby
-      gem 'redis-rails'
```

Comment:
`redis-rails` functionality is not supported out of the box by Rails, since v5.2, effectively deprecating this gem: https://github.com/redis-store/redis-rails#a-quick-note-about-rails-52